NuGet.Config

```diff
@@ -19,6 +19,7 @@
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="MyGetApplicationInsights" value="https://www.myget.org/F/applicationinsights" />
+    <add key="dotnet" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" protocolVersion="3" />
```

Comment:
Well, but I do want 2.4 because I want my EventSource telemetry module to target netstandard1.3. AFAIK the change for MS.AI.dll to target netstandard1.3 only went into 2.4 branch?